### ISSUE TITLE
Transformer based training details

### ISSUE THREAD

**Issue by luciencho** [chin-gyou/dialogue-utterance-rewriter#13]
Thanks for your paper and the code released. I am confused with some details about transformer-based training:
1. Did you try using the **pre-trained language model** to initialize encoder or decoder or just replace lstm with tranformer framework? Hints were in the code but not in the paper.
2. What are the missing hparams when you train transformer-based model? such like **warmup rate**, **decay rate**?
3. My summarization with LSTM without turn embedding, always decodes **duplicated words**, probably due to some word are more "important", does turn embedding play an important role in this problem?
